hieradata/role/jenkins-slave.yaml

```diff
@@ -0,0 +1,139 @@
+---
+psick::pre::windows_classes:
+  chocolatey: chocolatey
+psick::base::windows_classes:
+  packages: psick::packages
+  jenkins_slave: role::jenkins_slave
+psick::profiles::windows_classes:
+  features: psick::windows::features
+  registry: psick::windows::registry
+  services: psick::windows::services
+  puppet: psick::puppet::pe_agent
+  reboot: psick::reboot
+# FIXME: Why commented? Do we need them?
+#  firewall: windows_firewall
+#  users: psick::users
+
+psick::packages::resource_default_arguments:
+  provider: chocolatey
+psick::packages::packages_list:
+  - "chocolatey"
+  - "tortoisesvn"
+  - "putty.install"
+  - "git"
+  - "microsoft-build-tools"
```

Comment:
As discussed, the only Microsoft dev things we need are:`visualstudio2017enterprise visualstudio2017-workload-netweb`. Please remove `microsoft-build-tools netfx-4.6.1-devpack`. I think this is correct, we should try. Please also add `nodejs-lts`.